documentation/getting-started.md

```diff
@@ -35,3 +39,133 @@ this module wherever you'd like to use the components.
+to get everything up and running.
+
+### Step 1: Install the barista-components and Angular CDK
+
+Install the Barista components library via npm or yarn:
+
+`npm install --save @dynatrace/barista-components @angular/cdk highcharts@6 @types/highcharts @dynatrace/barista-icons d3-scale@^3.0.0 d3-shape@^1.3.5`  
+or  
+`yarn add @dynatrace/barista-components @angular/cdk highcharts@6 @types/highcharts @dynatrace/barista-icons d3-scale@^3.0.0 d3-shape@^1.3.5`
+
+### Step 2: Animations
+
+Some Barista components depend on the Angular animations module. If you want
+these animations to work in your app, you have to install the
+`@angular/animations` module and include the `BrowserAnimationsModule` in your
+app.
+
+`npm install --save @angular/animations`  
+or  
+`yarn add @angular/animations`
+
+```typescript
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+@NgModule({
+  ...
+  imports: [BrowserAnimationsModule],
+  ...
+})
+export class AppModule { }
+```
+
+If you don't want to add another dependency to your project, you can use the
+NoopAnimationsModule.
+
+```typescript
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+
+@NgModule({
+  ...
+  imports: [NoopAnimationsModule],
+  ...
+})
+export class AppModule { }
+```
+
+**Note:** @angular/animations uses the WebAnimation API that isn't supported by
+all browsers yet. If you want to support animations in these browsers, you'll
+have to
+[include a polyfill](https://github.com/web-animations/web-animations-js).
+
+### Step 3: Import the component modules
+
+Import the NgModule for each component you want to use:
+
+```typescript
+import { DtButtonModule } from '@dynatrace/barista-components/button';
+import { DtSelectModule } from '@dynatrace/barista-components/select';
+
+@NgModule({
+  ...
+  imports: [DtButtonModule, DtSelectModule],
+  ...
+})
+export class PizzaPartyAppModule { }
+```
+
+Alternatively, you can create a separate NgModule that imports all of the
+Barista components that you will use in your application. You can then include
+this module wherever you'd like to use the components.
+
+**Note:** Whichever approach you use, be sure to import the barista-components
+modules after Angular's BrowserModule, as the import order matters for
+NgModules.
+
+### Step 4: Include the styles
+
+The Barista components library ships with two different variants for including
+styles. You can either import the core styles that are **required** to use the
+barista-components. These core styles are just the bare minimum for the
+component and do not change the global styling of your app.
+
+```scss
+@import '~@dynatrace/barista-components/style/main';
+```
+
+You can also import the second variant, which include in addition to the core
+style also general styling for headlines, text-formatting, ...
+
+```scss
+@import '~@dynatrace/barista-components/style/index';
```

Comment:
add to angular json instead of import it inside the scss:

```json
   "styles": ["node_modules/@dynatrace/barista-components/style/index.scss"],
```